src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -104,6 +109,19 @@ const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
         {articleInformationTranslated}
       </VisuallyHiddenText>
       <ul css={BylineCss.bylineList} role="list">
+        {image ? (
```

Comment:
This can be changed to check if `image` is truthy, rather than a ternary that can return `null`

```suggestion
        {image && (
```